src/app/dataset-view/additional-components/metadata-component/components/form-components/input-field/input-field.component.ts

```diff
@@ -0,0 +1,14 @@
+import { ChangeDetectionStrategy, Component, Input } from "@angular/core";
+import { BaseField } from "../base-field";
+
+@Component({
+    selector: "app-input-field",
+    templateUrl: "./input-field.component.html",
+    styleUrls: ["./input-field.component.sass"],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class InputFieldComponent extends BaseField {
+    @Input() public value: string;
+    @Input() public placeholder?: string;
+    @Input() public requiredField?: boolean;
```

Comment:
Don't use undefined booleans, we only have 2 states - YES or NO, not 3rd UNDEFINED state. Instead have a default value.